docs/sphinx/source/api.rst

```diff
@@ -89,7 +89,6 @@ calculations.
    solarposition.equation_of_time_spencer71
    solarposition.equation_of_time_pvcdrom
    solarposition.hour_angle
-   solarposition.sun_rise_set_transit_geometric
```

Comment:
what's the motivation for removing this function here? Seems to me that it fits the section description.

Reply:
It's also listed in another section (`Functions for calculating sunrise, sunset and transit times`) and, to split hairs, it calculates times rather than positions.  Happy to put it back if you prefer, I don't feel strongly either way.

Reply:
fair enough!